nix-repl.el

```diff
@@ -0,0 +1,88 @@
+;;; nix-repl.el --- Nix repl
+;; -*- lexical-binding: t -*-
```

Comment:
This has to be on the first line to take effect. This file and all the other are still dynamically scoped.